Scripts/TimeConvert.cs

```diff
@@ -0,0 +1,48 @@
+ using System;
+
+ namespace SpacetimeDB
+ {
+     /// <summary>
+     /// Handles conversion between long (Rust Timestamps / microseconds since UNIX epoch)
+     /// and DateTimeOffset (UTC)
+     /// </summary>
+     public static class TimeConvert
+     {
+         /// <summary>
+         /// Converts from long (Rust Timestamps / microseconds since UNIX epoch) to DateTimeOffset (UTC) 
+         /// </summary>
+         public static DateTimeOffset FromMicrosecondsTimestamp(long microseconds)
+         {
+             long ticks = microseconds * 10;
+             return DateTime.UnixEpoch.AddTicks(ticks);
+         }
+
+         /// <summary>
+         /// Converts from DateTimeOffset (UTC) to long (Rust Timestamps / microsecondssince UNIX epoch)
+         /// </summary>
+         public static long ToMicrosecondsTimestamp(DateTimeOffset dateTimeOffset)
+         {
+             TimeSpan elapsed = dateTimeOffset - DateTime.UnixEpoch;
+             return elapsed.Ticks / 10;
+         }
+
+         /// <summary>
+         /// DateTimeOffset.Now (UTC) to long (Rust Timestamps / microseconds since UNIX epoch)
+         /// </summary>
+         public static long MicrosecondsTimestamp()
+         {
+             TimeSpan elapsed = DateTimeOffset.Now - DateTime.UnixEpoch;
+             return elapsed.Ticks / 10;
+         }
```

Comment:
Still not sure about the class itself and going to leave it to others to review, but another small nit is that we can reuse the more general method here:

```suggestion
         public static long MicrosecondsTimestamp() => ToMicrosecondsTimestamp(DateTimeOffset.Now);
```